06-best-practices/code/infrastructure/vars/prod.tfvars

```diff
@@ -0,0 +1,8 @@
+// Absolute path?
+lambda_function_local_path="../lambda_function.py"
```

Comment:
Why do we need it and the path to the dockerfile? Isn't just the registry URI sufficient?

Reply:
The lambda infra in TF fails if the config is expecting an Image source and URI, but if the URI doesn't exist. So we either bootstrap a dummy image in the ECR repo to link the lambda with, or we use this one and then update the environment vars in the CI/CD pipeline